pkg/responseerrors/responseerrors.go

```diff
@@ -9,14 +9,16 @@ import (
 
 var (
 	// #nosec G101 This is a constant error code, no security risk.
-	notBearerTokenFormatErrorCode = "ERR_TOKEN_NOT_BEARER_FORMAT"
-	missingBearerTokenErrCode     = "ERR_MISSING_BEARER_TOKEN"
-	defaultNotFoundErrCode        = "ERR_NOT_FOUND"
-	defaultForbiddenErrCode       = "ERR_FORBIDDEN"
-	defaultUnauthorizedErrCode    = "ERR_UNAUTHORIZED"
-	methodNotAllowedErrCode       = "ERR_METHOD_NOT_ALLOWED"
-	invalidResourceIDErrCode      = "ERR_INVALID_RESOURCE_ID"
-	unknownErrCode                = "ERR_UNKNOWN"
+	notBearerTokenFormatErrorCode     = "ERR_TOKEN_NOT_BEARER_FORMAT"
```

Comment:
I would rather structure the error codes as `errCodeX` rather than `xErrorCode`, this makes it easier to grep them both visually and via autocomplete.